### PR TITLE
SDIT-1505 Integration with alerts DPS APIs

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsDpsApiMockServer.kt [hunk 1]
```diff
@@ -32,6 +40,25 @@ class AlertsDpsApiMockServer : WireMockServer(WIREMOCK_PORT) {
...
+      alertSeq = 2,
+      alertUuid = UUID.randomUUID(),
+      status = CREATED,
+    ),
+  ) {
+    stubFor(
+      post("/alerts")
+        .willReturn(
+          aResponse()
+            .withStatus(200)
```
Comment (mikehalmamoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsDpsApiMockServer.kt:55]: If their endpoint is an upsert then a 200 would make sense - or is it a 201 for create and 200 for update?

Comment (iterative-it) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsDpsApiMockServer.kt:55]: Yes it does use 201 to indicate that the request created an alert in the new service and 200 when an existing alert was updated instead

Comment (andymarke) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsDpsApiMockServer.kt:55]: thanks

Comment (andymarke) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsDpsApiMockServer.kt:55]: BTW @iterative-it I couldn't see the response code in the swagger (not that it matters so long as it is in the 2xx range)

**Pull request merged**